scripts/choose-test-instance.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Do you know [shellcheck](https://github.com/koalaman/shellcheck)? It's a tool to check common errors in shell scripts and make them more robust.

I also think that using `set -eu` for this script could be safer. See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ for an explanation.